Userland/Libraries/LibCore/FileSystem.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2023, Cameron Youell <cameronyouell@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/DeprecatedString.h>
+#include <AK/Error.h>
+#include <AK/StringView.h>
+#include <LibCore/File.h>
+#include <sys/stat.h>
+
+namespace Core::FileSystem {
+
+#define DEFAULT_PATH "/usr/local/sbin:/usr/local/bin:/usr/bin:/bin"
```

Comment:
If it isn't entirely out of scope, can we not have `DEFAULT_PATH` as preprocessor defines?

`static constexpr StringView default_path = "/usr/local/sbin:/usr/local/bin:/usr/bin:/bin"sv` will work fine for all `DEFAULT_PATH_SV` uses, and the few occurences in LibC/unistd.cpp and TelnetServer/main.cpp can use `.characters()`